sinon/lib/util/SpyCall.py

```diff
@@ -10,16 +10,16 @@ class SpyCall(object):
     Holds data and test functions related to a single function call
     """
 
-    next_spy_call_id = 0
+    __next_spy_call_id = 0
```

Comment:
Same as the above reason of `__get_wrapper(self)`, I suggest to change to **protected** variable, since it will be used.